lib/versioning/rez/readme.md

```diff
@@ -0,0 +1,16 @@
+rez versioning was developed to support the [rez Package Manager](https://github.com/nerdvegas/rez).
+It's based on Semantic versioning but includes its own concept of ranges.
+
+**Use of dots**
+
+A range is expressed with dots `1.2..2` means `>=1.2.x <2.0.0`.
+
+**No exact versions unless using two equals ==**
+
+In rez, `1.2.3` doesn't mean "exactly 1.2.3", it actually means `>= 1.2.3 <1.2.4`.
+If you want to use an exact version use two equal characters, like this: `==1.2.3`.
+
+**Use of pipes**
```

Comment:
Do we need an explanation of what `pipes` are used for in rez, or is this obvious to people who use `rez`?
From reading the current doc we have here, it's not clear what functionality I'm missing because we lack pipe support.

Reply:
How about adding something like this? Always better to make things easier for a new reader.

```suggestion
**Use of pipes**

rez uses pipes as an OR operator, "2.7..3|4" means ">=2.7 <3 OR 4.x.x".
```

Reply:
I like it! 👍